app/app.go

```diff
@@ -414,7 +424,7 @@ func NewEvmos(
 	// use custom Ethermint account for contracts
 	app.AccountKeeper = authkeeper.NewAccountKeeper(
 		appCodec, keys[authtypes.StoreKey],
-		evmostypes.ProtoAccount, maccPerms,
+		authtypes.ProtoBaseAccount, maccPerms,
```

Comment:
We're still using the EthAccounts
```suggestion
		evmostypes.ProtoAccount, maccPerms,
```